### PR TITLE
feat: duration

### PR DESCRIPTION
**Pull request by wingkwong**
### Support formats

image

h:mm
image

h:mm:ss
image

h:mm:ss.s
image

h:mm:ss.ss
image

h:mm:ss.sss
image

## Change type

- [x] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

### CODE REVIEW

File: packages/nc-gui/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "jsep": "^1.3.6",
     "locale-codes": "^1.3.1",
     "material-design-icons-iconfont": "^5.0.1",
+    "moment": "^2.29.3",
```
Comment (o1lab) [inline packages/nc-gui/package.json:31]: Screenshot 2022-06-05 at 09 36 17

Moment is quite heavy. I guess we'd used an alternative. Please check.

Comment (karishmashuklaa) [reply packages/nc-gui/package.json:31]: @wingkwong We already have <a href="https://github.com/nocodb/nocodb/blob/develop/packages/nc-gui/package.json">dayjs</a> as a dependency

Comment (wingkwong) [reply packages/nc-gui/package.json:31]: @karishmashuklaa You're right. Will switch to dayjs then.

Comment (wingkwong) [reply packages/nc-gui/package.json:31]: @karishmashuklaa dayjs may not a way to go as it returns `NaN` if we input `minutes: 65`, while momentjs would return `hours: 1` and  `minutes: 5` which is expected. Will think about it.

Comment (wingkwong) [reply packages/nc-gui/package.json:31]: Let's ditch both

File: packages/nc-gui/helpers/durationHelper.js [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+// pad zero
+// mm && ss
+// e.g.  3 -> 03
+// e.g. 12 -> 12
+// sss
+// e.g.  1 -> 001
+// e.g. 10 -> 010
+const padZero = (val, isSSS = false) => {
+  if (isSSS) {
```
Comment (pranavxc) [inline packages/nc-gui/helpers/durationHelper.js:38]: you can simplify it by using 

```js
return `00${val}`.slice(isSSS ? -3 : -2)
```

or by using `padStart`

```js
return (val + '' ).padStart(isSSS ? 3 : 2, '0')
```

**Comment by wingkwong** [comment]
Closing as it includes lots of unnecessary commits. Will create a new one.

**Pull request closed without merging**